source/features/view-markdown-source.tsx

```diff
@@ -5,9 +5,75 @@ import delegate from 'delegate-it';
 import CodeIcon from 'octicon/code.svg';
 import FileIcon from 'octicon/file.svg';
 import * as pageDetect from 'github-url-detection';
+import KebabHorizontalIcon from 'octicon/kebab-horizontal.svg';
 
 import features from '.';
 import fetchDom from '../helpers/fetch-dom';
+import GitHubURL from '../github-helpers/github-url';
+import {getRepoURL} from '../github-helpers';
+
+function lineActions(): JSX.Element {
+	const blame = new GitHubURL(location.href).assign({route: 'blame'}).toString();
+	return (
+		<details
+			className="details-reset details-overlay BlobToolbar position-absolute js-file-line-actions dropdown d-none"
+			aria-hidden="true"
+		>
+			<summary
+				className="btn-octicon ml-0 px-2 p-0 bg-white border border-gray-dark rounded-1"
+				aria-label="Inline file action toolbar"
+				aria-haspopup="menu"
+				role="button"
+			>
+				<KebabHorizontalIcon/>
+			</summary>
+			<details-menu role="menu">
+				<ul
+					className="BlobToolbar-dropdown dropdown-menu dropdown-menu-se mt-2"
+					style={{width: '185px'}}
+				>
+					<li>
+						<clipboard-copy
+							role="menuitem"
+							className="dropdown-item"
+							id="js-copy-lines"
+							style={{cursor: 'pointer'}}
+						>Copy line
+						</clipboard-copy>
+					</li>
+					<li>
+						<clipboard-copy
+							role="menuitem"
+							className="dropdown-item"
+							id="js-copy-permalink"
+							style={{cursor: 'pointer'}}
```

Comment:
If this is required, just add `clipboard-copy {cursor: pointer}` to `refined-github.css`
```suggestion
```